js/blocks.js

```diff
@@ -4472,6 +4478,17 @@ class Blocks {
             }
         };
 
+        this._isConnectedToNoteValue = (blk) => {
+            if (NOTEBLOCKS.indexOf(this.blockList[blk].name) !== -1) {
```

Comment:
Remind me again what NOTEBLOCKS is?

Reply:
array containing name of notevalue block.